Pod density automation script

paigerube14 opened the pull request:
This PR will add in 2 different automation scripts for the pod_density test case. The first is a typical pod_density test run that will scale the cluster up to 20 worker nodes and uses cluster loader to load the content/pause_template.yaml template file. 
At the end of cluster loader finishing it checks to see if any pods have errors and spits out how many pods were put on each node (not sure if this is necessary)

The second script does more of a gradual increase in the number of pods it creates. It uses both the pod_density and cluster loader scripts to create a certain amount of projects and then slowly increase by a number you set.

mffiedler commented:
@skordas PTAL

mffiedler commented:
We don't have the greatest organization in svt, but we do have a convention of putting specific scale tests under openshift_scalability/content.   If it's not too much of a pain, let's move it their.   Might have to adjust relative dirs, I haven't looked in detail yet.

paigerube14 commented:
I updated the pod density scripts to live in a folder under scripts. Moved the yaml file to run into the content folder. Thoughts on this layout of files?

paigerube14 commented on openshift_scalability/scripts/pod_density/pod_density-README.md:
````diff
@@ -0,0 +1,44 @@
+### Install [pytimeparse](https://github.com/wroberts/pytimeparse) module
+
+```sh
+$ pip install -r cluster_loader_requirements.txt
````
Sorry missed adding this file originally. I have it at the top level openshift_scalability folder, should be added now (https://github.com/openshift/svt/blob/65f75eeffc28240707e826cc6fdfd7ebbcc9c366/openshift_scalability/cluster_loader_requirements.txt)

Need to update the reference in the readme

paigerube14 commented:
@skordas are you able to take a second look through this PR? I think I edited everything you pointed out and I also changed the pod_density readme to just be named README.md to be able to be seen at the folder level

skordas commented:
/lgtm
